src/api/export.rs

```diff
@@ -0,0 +1,28 @@
+//! provides common types used in the crate's public API
+//!
+//! They are exported here in one location for caller convenience.
```

Comment:
I find it confusing to mention the “caller.” Usually, documentation talks to said caller directly or doesn't mention them at all. For example, the [std prelude](https://doc.rust-lang.org/std/prelude/index.html) explicitly talks to “you”, the person interested in using the module, the [std::io prelude](https://doc.rust-lang.org/std/io/prelude/index.html) only describes what the module is for, the [proptest prelude](https://docs.rs/proptest/latest/proptest/prelude/index.html) also mentions what “you” can (or don't have to) do.

Reply:
ok, will change.

Reply:
addressed in 5a541c5c56ee98b81951620cababd4255bfc8df9